### DISCUSSION TITLE
Come si disabilitano i link con Tor

### DISCUSSION THREAD

**Discussion by alongo71** [globaleaks/globaleaks-whistleblowing-software#3976] [Q&A]
Salve a tutti, ho da poco installato il software, ma nelle notifiche email vengono inviati i link con Tor. Come si può disabilitare questa opzione e passare a link web?
 Copi e incollo un esempio:

_Gentile xxxxxxxxx

Con la presente email ti notifichiamo la ricezione di una nuova segnalazione.

La segnalazione può essere consultata accedendo all'indirizzo:
**http://5i7q4mzfqivwrbpl66zco74n2fqpvzayyp3r3eia7ewxlh6scyft5syd.onion/#/reports/3ced6ecb-5e3a-4bfe-a64d-11f8102ca4f0**

Distinti saluti,
xxxxxxxx_

Non si può far in modo che tutti questi link siano URL web normali?
Grazie

**Comment by evilaliv3** [comment]
Probabilmene non avete ancora abilitato un dominio HTTPS, è possibile?

Dal pannello amministrativo potete farlo accedendo alla configurazione di rete.

Dopo che avrete configurato un dominio HTTPS tradizionale nelle email verranno generati indirizzi sulla base di quello.

**Comment by alongo71** [comment]
Si ...in effetti è così...non abbiamo ancora abilitato un domino HTTPS. Grazie mille per la prona risposta.
